schemas/logs/salesforce/insecure-external-assets.yml

```diff
@@ -0,0 +1,145 @@
+schema: Salesforce.InsecureExternalAssets
+version: 0
+referenceURL: https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_objects_eventlogfile_insecureexternalassets.htm
+description: >-
+  Insecure External Assets events contain information about external assets.
+  External assets include images or videos accessed by users over an insecure HTTP protocol.
+  The event lists all your Salesforce pages that contain assets hosted insecurely on third-party sites
+  that users loaded with a Chrome, Firefox, Microsoft Edge, or Safari browser.
+parser:
+  csv:
```

Comment:
Just double-checking: How do empty values look at Salesforce logs? Do we need to specify a special character as `emptyValues`? https://docs.runpanther.io/data-onboarding/custom-log-types/reference#parser-csv-fields

Reply:
I haven't encountered or read anything related in the documentation. Empty values are empty strings `,"",` as far as I have observed.